splat.yaml

```diff
@@ -21,6 +21,8 @@ options:
   disassemble_all: False
   # Allow use of 64-bit floating point registers
   generated_s_preamble: .set fp=64
+  asm_nonmatching_label_macro : ""
+  asm_data_end_label: ""
```

Comment:
This isn't necessary, the current splat version we have pinned automatically generates include/macro.inc now which adds the necessary macros to handle spimdisasm's new nonmatching labels. I like the idea of having those new .NONMATCHING symbols, so I'd rather not turn it off (could be useful to simplify or make the progress calc more accurate? haven't played around with it yet)

If you're having issues, try upgrading your python packages `pip install -r requirements.txt --upgrade`